scripts/Muon/GUI/Common/plot_widget/plot_widget_presenter.py

```diff
@@ -116,11 +116,13 @@ def handle_plot_mode_changed_for_muon_analysis(self, plot_mode : PlotMode):
         self._view.set_plot_mode(str(plot_mode))
         if plot_mode == PlotMode.Data:
             self._view.enable_plot_type_combo()
+            self._view.plot_diff_checkbox.setVisible(False)
             self.update_plot()
             self.fitting_plot_range = self._figure_presenter.get_plot_x_range()
             self._figure_presenter.set_plot_range(self.data_plot_range)
         elif plot_mode == PlotMode.Fitting:
             self._view.disable_plot_type_combo()
+            self._view.plot_diff_checkbox.setVisible(True)
```

Comment:
Sorry I should have raised this before, but its only a minor issue. But can we implement a method similar to
disable_plot_type_combo and enable_plot_type_combo, within the [view ](https://github.com/mantidproject/mantid/blob/69a834894f26b22df9d14d5ef77024e934dc31c6/scripts/Muon/GUI/Common/plot_widget/plot_widget_view.py#L25) and the [view interface.](https://github.com/mantidproject/mantid/blob/69a834894f26b22df9d14d5ef77024e934dc31c6/scripts/Muon/GUI/Common/plot_widget/plot_widget_view_interface.py) The presenter here doesn't need to know that its dealing with a qt checkbox. Then we can replace the calls here with the new method.